src/utilities/templates/flatpages/base.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base_sidebar.html" %}
```

Comment:
This base template file should match the base template file in Announcements, Badges, and everywhere else, with contextual changes as needed.

Reply:
Fixed: Copied the format of other base.html, only difference was I stripped the block tags of that were never used